reporter/src/main/scala/com/typesafe/tools/mima/lib/analyze/method/MethodChecker.scala

```diff
@@ -37,7 +37,9 @@ private[analyze] abstract class BaseMethodChecker extends Checker[MemberInfo, Cl
 
 private[analyze] class ClassMethodChecker extends BaseMethodChecker {
   def check(method: MemberInfo, inclazz: ClassInfo): Option[Problem] = {
-    if (method.isDeferred)
+    if (method.nonBridge && method.nonAccessible)
```

Comment:
So this depends on if you want MiMa to report the same things that it does right now. Right now (before looking at `ACC_SYNTHETIC`) it reports the erased apply method as a binary incompatibility as well as the normal apply method. The erased apply method is marked `ACC_SYNTHETIC | ACC_BRIDGE` as per the JLS **Example 15.12.4.5-1. Invoked Method Signature Has Different Erasure Than Compile- Time Method Signature**. We can of course skip reporting this erased apply method as well, since it always lives with a normal apply method that will be reported.